### DISCUSSION TITLE
[不具合] 編集中のコメントが画面をスクロールすると消えてしまう

### DISCUSSION THREAD

**Discussion by ko1nksm** [increments/qiita-discussions#603] [要望・アイデア・不具合]
記事に対してコメントをした後、そのコメントの内容を修正している途中（保存しない）に、画面をある程度スクロールして上の方を表示してから戻ると、編集中のコメントの内容が消えて元に戻ってしまうようです。

**Comment by tanaka-asami415** [comment]
@ko1nksm 
ご報告ありがとうございます！
不具合の可能性が高いため、一旦調査します🙇

**Comment by kimkim0814** [comment]
@ko1nksm
この度はご報告頂きありがとうございます!

頂いた内容に関しまして調査を行ったところ、
コメントフォームの不具合であることが判明したため修正します :bow: :sweat_drops:

修正完了するまでは、ご不便おかけいたしますが
よろしくお願いいたします。

**Comment by ohakutsu** [comment]
@ko1nksm
大変おまたせしました！
こちらの不具合の修正が完了しました！

**Reply by ko1nksm** [reply]
ありがとうございます！（結構前から修正されていたような？）

**Reply by ohakutsu** [reply]
ご確認ありがとうございます！
（申し訳ございません、以前修正した際にご連絡が漏れておりました :bow:）
